raspberry-pi/4/apply-overlays-dtmerge.nix

```diff
@@ -0,0 +1,30 @@
+# modification of nixpkgs deviceTree.applyOverlays to resolve https://github.com/NixOS/nixpkgs/issues/125354
```

Comment:
Its currently not clear to me where this file is used in the code. Also if it fixes an issue in the nixpkgs version, can we just merge into nixpkgs directly?

Reply:
Ah, it's used in the other PR: https://github.com/NixOS/nixos-hardware/pull/442

Reply:
So this needs to be removed now.

Reply:
yup

Reply:
Done